### PR TITLE
Update init.lua

### PR DESCRIPTION
**Pull request by pierrenel**
Added an option to remove current line from the bookmarks db (if it's set) via leader bd / MinimalBookmarksDelete

### CODE REVIEW

File: lua/minimal_bookmarks/init.lua [hunk 1]
```diff
@@ -162,6 +162,91 @@ local function add_bookmark(append, name)
     add_current_line_to_bookmarks(append, name)
 end
 
+
+local function is_current_line_bookmarked()
```
Comment (yuriescl) [inline lua/minimal_bookmarks/init.lua]: What is the purpose of this function? I don't see it being used anywhere

File: lua/minimal_bookmarks/init.lua [hunk 2]
```diff
@@ -162,6 +162,91 @@ local function add_bookmark(append, name)
...
+                return true
+            end
+        end
+    end
+    
+    file:close()
+    return false
+end
+
+local function delete_current_line_bookmark()
```
Comment (yuriescl) [inline lua/minimal_bookmarks/init.lua:165]: This function looks good, ready for approval. Just need clarification on the other function.

**Review by yuriescl — CHANGES_REQUESTED** [review]
Looks good except need clarification on a function

File: lua/minimal_bookmarks/init.lua (hunk 1 above)
Comment (pierrenel) [reply lua/minimal_bookmarks/init.lua]: Aaah I was working on a thing to display something in the gutter, I'll remove it for now :)

**Comment by yuriescl** [comment]
Thank you for your contribution

**Pull request merged**